Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:14-buster-slim
+
+WORKDIR /app
+
+COPY package.json package-lock.json ./
+
+RUN npm install --production
+
+COPY layouts layouts
```

Comment:
This is redundant with the `.deployinclude` file. Another way is to use a `.dockerignore` and instead list what we shouldn't deploy. I don't think it matters a lot since we'll delete `deploy.sh` and deploy this with something `doctl` commands from GitHub Actions, most likely.